package.json

```
@@ -35,19 +35,21 @@
     "install-if-deps-outdated": "node bin/install-if-deps-outdated.js",
     "lint": "npm run lint:js && npm run lint:css",
     "lint:css": "stylelint 'client/**/*.scss' 'client/**/*.css' 'assets/**/*.scss' 'assets/**/*.css'",
-    "lint:css-fix": "npm run lint:css -- --fix",
     "lint:js": "eslint . --ext=js,jsx",
     "lint:php": "./vendor/bin/phpcs --standard=phpcs.xml.dist $(git ls-files | grep .php$)",
-    "lint:php-fix": "./vendor/bin/phpbf --standard=phpcs.xml.dist $(git ls-files | grep .php$)"
+    "lint:php-fix": "./vendor/bin/phpbf --standard=phpcs.xml.dist $(git ls-files | grep .php$)",
+    "format": "npm run format:js && npm run format:css",
+    "format:js": "npm run format:provided 'assets/**/*.js' 'client/**/*.js'",
+    "format:css": "npm run format:provided 'client/**/*.scss' 'assets/**/*.css'",
+    "format:provided": "prettier --write"
   },
   "husky": {
     "hooks": {
       "pre-commit": "lint-staged"
     }
   },
   "lint-staged": {
-    "*.{js,jsx}": "npm run lint:js",
-    "*.scss": "npm run lint:css",
+    "*.{js,jsx,scss,css}": "npm run format:provided",
```

`eslint` can catch some mistakes that `prettier` can't. This should be:

```suggestion
    "*.{js,jsx}": ["npm run format:provided", "eslint"],
    "*.{scss,css}": ["npm run format:provided", "stylelint"],
```

To reproduce, declare a variable in JS but never use it. That's an error that only `eslint` can catch. That should fail the pre-commit hook.

---

I'd suggest following changes on top of @DanReyLop 's suggestions:

* `eslint` => `npm run lint:js`
* `stylelint` => `npm run lint:css`

---

`npm run lint:js` runs the linter on all the files though, that's inefficient. Same for `npm run lint:css`. We just need to run them on the "staged" files on the pre-commit hook.

---

AFAIK lint-staged handles that.

---

Made local test and confirm it works like that:

1. Change two js files. Add unused const to one of them.
2. Stage only correct file.
3. Commit changes.

Result: commited successfully, unstaged file does not affect commit.

---

Added linting in 774e74f